README.md

```diff
@@ -166,6 +166,16 @@ If you Watch this repository, GitHub will send you an email every time I publish
         <td>0dp</td>
         <td>sets elevation to ticket view on android jellybean and above</td>
     </tr>
+    <tr>
+        <td>app:ticketBackgroundBelowDivider</td>
+         <td>none</td>
+         <td>sets background to ticket view below divider on android jellybean and above</td>
```

Comment:
The statement over here is slightly wrong, this drawable can be applied on any android version.

Reply:
fixed in both